README.md

```diff
@@ -64,17 +64,17 @@ These instructions will have the code running on your local or virtual machine.
 
 You will need the following to have a working copy of this software. See
 [installation](#installation-instructions) steps:
-1. Python (>=3.7.9)
+1. Python (>=3.7) ((originally stated "3.7.9", it ran at "3.7.5". My local machine runs 3.8.8)
```

Comment:
For cleanliness, recommend moving the comment to the pull request message instead.
```suggestion
1. Python (>=3.7) 
```